addons/casings/initSettings.sqf

```diff
@@ -0,0 +1,19 @@
+[
+    QGVAR(enabled), "CHECKBOX",
+    [LSTRING(displayName), LSTRING(description)],
+    LSTRING(Settings_DisplayName),
+    true,
+    true,
+    {},
+    true
```

Comment:
As this spawns objects in the end, it can be a performance hog for older system, I would suggest that this should be a client setting.